config/flags.m4

```diff
@@ -22,7 +22,7 @@ AC_DEFUN([SET_AM_CFLAGS],
   AM_CFLAGS=${AM_CFLAGS}" -fPIC "
   
   # Hidding all unneeded symbols.
-  AM_CFLAGS=${AM_CFLAGS}" -fvisibility=hidden "
+  #AM_CFLAGS=${AM_CFLAGS}" -fvisibility=hidden "
```

Comment:
Why has this been removed?

Reply:
Some symbols were not visible when building the shared library.
On make test I was getting following errors:
Undefined symbols for architecture x86_64:
  "_OQS_KEM_frodokem_640_aes_decaps", referenced from:
      _example_stack in example_kem.o
  "_OQS_KEM_frodokem_640_aes_encaps", referenced from:
      _example_stack in example_kem.o
  "_OQS_KEM_frodokem_640_aes_keypair", referenced from:
      _example_stack in example_kem.o
  "_OQS_MEM_cleanse", referenced from:
      _example_stack in example_kem.o
      _cleanup_stack in example_kem.o
  "_OQS_MEM_insecure_free", referenced from:
      _example_heap in example_kem.o
      _cleanup_heap in example_kem.o
  "_OQS_MEM_secure_free", referenced from:
      _example_heap in example_kem.o
      _cleanup_heap in example_kem.o

Reply:
Can you investigate to see the cause of the individual errors?  If memory serves we turned `-fvisibility=hidden` on for a reason, to make sure we were exporting exactly the symbols we wanted to export.  So if that's not the case, I'd rather fix it than just turn the feature off.